src/org/kernelci/debian/rootfs.groovy

```diff
@@ -32,25 +32,16 @@ KCI_TOKEN_ID
 - Harcoded paths at jenkins/debian/... for Dockerfile and debos file
 */
 
-def call(Closure context) {
+package org.kernelci.debian
 
-    def config = [:]
-    context.resolveStrategy = Closure.DELEGATE_FIRST
-    context.delegate = config
-    context()
+
+def buildImage(config) {
```

Comment:
Is it really better to pass a `config` object than 3 arguments?  How about this instead:

```Groovy
def buildImage(name, archList, debianSuite) {
```

Reply:
Yeah, this is on purpose. In the future, we might want to add new parameters to build the image without needing to update all the *.jpl files.